packages/rolldown/src/watcher.ts

```diff
@@ -42,7 +42,15 @@ export class Watcher {
         break
       case 'event':
         this.inner.on(BindingWatcherEvent.Event, async (data) => {
-          await listener(data)
+          if (data!.code === 'BUNDLE_END') {
+            await listener({
+              code: 'BUNDLE_END',
+              duration: Number(data!.duration),
```

Comment:
Why not just output `number` rust side?

Reply:
Because the data typing is ` WatcherEventData(Option<HashMap<String, String>>)`, binding `HashMap<String, String/Number>` is complex. 
I have tried to convert it to JSON string and parse it at js side, but it is also not good idea.